config/helm/aws-node-termination-handler/values.yaml

```diff
@@ -213,41 +218,50 @@ podMonitor:
   # The number of scraped samples that will be accepted
   sampleLimit: 5000
 
-serviceMonitor:
-  # Specifies whether ServiceMonitor should be created
-  # this needs enableSqsTerminationDraining: true
-  # and enablePrometheusServer: true
-  create: false
-  # Specifies whether the ServiceMonitor should be created in a different namespace than
-  # the Helm release
-  namespace:
-  # Additional labels to add to the metadata
-  labels: {}
-  # The Prometheus scrape interval
-  interval: 30s
-  # The number of scraped samples that will be accepted
-  sampleLimit: 5000
-
-# K8s DaemonSet update strategy.
-updateStrategy:
-  type: RollingUpdate
-  rollingUpdate:
-    maxUnavailable: 1
-linuxUpdateStrategy: ""
-windowsUpdateStrategy: ""
-
 # Determines if NTH uses host networking for Linux when running the DaemonSet (only IMDS mode; queue-processor never runs with host networking)
 # If you have disabled IMDSv1 and are relying on IMDSv2, you'll need to increase the IP hop count to 2 before switching this to false
 # https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/configuring-instance-metadata-service.html
 useHostNetwork: true
 
-# The maximal amount of parallel event processors to handle concurrent events
-workers: 10
+# Daemonset DNS policy
+dnsPolicy: ""
+linuxDnsPolicy: ""
+windowsDnsPolicy: ""
 
-# The number of replicas in the NTH deployment when using queue-processor mode (NOTE: increasing this may cause duplicate webhooks since NTH pods are stateless)
-replicas: 1
+linuxNodeSelector: {}
+windowsNodeSelector: {}
 
-# podDisruptionBudget specifies the disruption budget for the controller pods.
-# Disruption budget will be configured only when the replicaCount is greater than 1
-podDisruptionBudget: {}
-#  maxUnavailable: 1
+daemonsetAffinity:
+  nodeAffinity:
+    requiredDuringSchedulingIgnoredDuringExecution:
+      nodeSelectorTerms:
+        - matchExpressions:
+            - key: "eks.amazonaws.com/compute-type"
+              operator: NotIn
+              values:
+                - fargate
+linuxAffinity: {}
+windowsAffinity: {}
+
+daemonsetTolerations:
```

Comment:
The `daemonsetTolerations` replaces `tolerations` as previously `tolerations` were configured for the daemonset and wouldn't be right for the deployment. Now `tolerations` will win if they're set by the user, for backwards compatibility, but we could make the chart simpler by "breaking" this a bit more and not allowing `tolerations` to apply to the daemonset?

Reply:
Gotcha, I think the way you have it is fine. I'd like to keep as much backwards compatibility as possible until we release a v2 since a lot of folks installed NTH a while ago, and maybe have upgraded occasionally, but I'd hate to have NTH be a nuisance to manage and upgrade. Especially for those opting for IMDS mode which should be super simple.